docs/snapshots.md

```diff
@@ -23,32 +21,22 @@ Technical documentation can be found [here](https://pkg.go.dev/github.com/dgraph
 
 Index snapshots are created using the CLI tool found at `cmd/create-index-snapshot`.
 Usage for the tool is described in more detail [here](https://github.com/optakt/flow-dps/blob/master/cmd/create-index-snapshot/README.md).
-By default, the snapshot is hex-encoded.
+By default, the snapshot is not encoded and output contains raw (binary) data.
 
 ```console
-$ create-index-snapshot -i <index_dir> > output.hex
+$ create-index-snapshot -i <index_dir> > output.bin
 ```
 
-When an index snapshot is created, it is compressed using a specific compression dictionary.
-When restoring the index, the snapshot needs to be decompressed using the same dictionary or decompression will fail.
+When an index snapshot is created, it can be compressed with a specific compression algorithm (zstd or gzip).
+When restoring the index, the snapshot needs to be decompressed using the same algorithm or snapshot restore will fail.
 
-## Updating a Snapshot
+## Restoring a Snapshot
 
-It is sometimes necessary to regenerate the snapshot, for example when an index was added or changed, or because the internal format of the index changed (like when a compression dictionary is changed).
-
-### New Index
-
-The following command creates a new index.
-Be careful to always use the same inputs as for the original index, if the goal is to continue building on the existing state.
-
-```console
-$ flow-dps-indexer -t <trie_dir> -d <data_dir> -i <index_dir> -a
-```
-
-### New Snapshot
-
-Creating the new snapshot is the same as described in the [Creating a snapshot](#creating-a-snapshot) section, while referencing the new index.
+To restore a snapshot, use the CLI tool found at `cmd/restore-index-snapshot`.
+Usage for the tool is described in more detail [here](https://github.com/optakt/flow-dps/blob/master/cmd/restore-index-snapshot/README.md).
```

Comment:
Restoring snapshots is done using the `restore-index-snapshot` binary, which is documented [here](https://github.com/optakt/flow-dps/blob/master/cmd/restore-index-snapshot/README.md).